config/serverless-operator.yaml

```diff
@@ -71,3 +66,197 @@ repositories:
+        releaseBuildConfiguration:
+          base_images:
+            hypershift-operator:
+              name: "4.13"
+              namespace: ocp
+              tag: hypershift-operator
+            upi-installer:
+              name: "4.13"
+              namespace: ocp
+              tag: upi-installer
+          releases:
+            latest:
+              integration:
+                include_built_images: true
+                name: "4.13"
+                namespace: ocp
+          tests:
+            - as: e2e-hypershift-continuous
+              cron: 0 0 * * 0
+              steps:
+                cluster_profile: aws-cspi-qe
+                env:
+                  BASE_DOMAIN: cspilp.interop.ccitredhat.com
+                  HYPERSHIFT_BASE_DOMAIN: cspilp.interop.ccitredhat.com
+                  HYPERSHIFT_NODE_COUNT: "4"
+                test:
+                  - as: operator-e2e
+                    commands: make USER_MANAGEMENT_ALLOWED=false test-e2e-with-kafka
+                    # dependencies: Generated by openshift-knative/hack
+                    from: serverless-source-image
+                    resources:
+                      limits:
+                        memory: 6Gi
+                      requests:
+                        cpu: 100m
+                        memory: 200Mi
+                  - as: knative-serving-eventing-e2e
+                    commands: make test-upstream-e2e-no-upgrade
+                    # dependencies: Generated by openshift-knative/hack
+                    from: serverless-source-image
+                    resources:
+                      limits:
+                        memory: 6Gi
+                      requests:
+                        cpu: 100m
+                        memory: 200Mi
+                  - as: knative-eventing-kafka-broker-e2e
+                    commands: make test-upstream-e2e-kafka-no-upgrade
+                    # dependencies: Generated by openshift-knative/hack
+                    from: serverless-source-image
+                    resources:
+                      limits:
+                        memory: 6Gi
+                      requests:
+                        cpu: 100m
+                        memory: 200Mi
+                workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
+              timeout: 8h0m0s
+      - name: "osd" # Results in openshift-knative-serverless-operator-<BRANCH>__<OCP_VERSION>-osd.yaml
+        releaseBuildConfiguration:
+          base_images:
+            cli-ocm:
+              name: cli-ocm
+              namespace: ci
+              tag: latest
+          tests:
+            - as: e2e-osd-continuous
+              cron: 0 0 * * 0
+              steps:
+                cluster_profile: osd-ephemeral
+                env:
+                  CLUSTER_DURATION: "10800"
+                  CLUSTER_VERSION: "4.13"
```

Comment:
If this generates 2 files like 
```
openshift-knative-serverless-operator-main__413-osd.yaml
openshift-knative-serverless-operator-main__411-osd.yaml
```

shouldn't `env.CLUSTER_VERION` be based on the actual osd version ?

Reply:
It is only generated for one version due to this: https://github.com/openshift-knative/hack/pull/96/files#diff-8c104313544b2c9a8413d2cbb7a399e56793ffab4a3e554d5ac2243a5a74f1c8R10
I'm describing the whole issue in https://github.com/openshift-knative/hack/pull/96/files#r1420289826

We'd need to add some kind of generating this.